### PR TITLE
Inno6 installer

### PR DESCRIPTION
**Pull request by acidcoke**
@willwade I took the liberty to create a fork of gameface since I was unhappy with googles way of handling projects. I hope you are okay with this PR.

### CODE REVIEW

**Comment by willwade** [comment]
We have an ev signing certificate. Happy to sign it - it's not doable under GH Actions to sign but on release we can

**Comment by acidcoke** [comment]
That sounds great. I have never worked with implementing installers before so I will take some time to properly review it.

**Comment by willwade** [comment]
It's pretty easy. There's inno6 or nsis. I went with inno6 for this only because I was having a headache with file paths. Note the big change to the source was putting file path locations in readable places under windows 10/11. It's not reliable now to put stuff just in program files.

File: src/shape_list.py [hunk 1]
```diff
@@ -85,9 +85,7 @@
     "Open mouth": "assets/images/dropdowns/Open mouth.png",
     "Mouth left": "assets/images/dropdowns/Mouth left.png",
     "Mouth right": "assets/images/dropdowns/Mouth right.png",
-    "Roll lower mouth": "assets/images/dropdowns/Roll lower mouth.png",
     "Raise left eyebrow": "assets/images/dropdowns/Raise left eyebrow.png",
-    "Lower left eyebrow": "assets/images/dropdowns/Lower left eyebrow.png",
```
Comment (acidcoke) [inline src/shape_list.py:90]: Can you please add these 2 gestures back in?

File: README.md
Comment (acidcoke) [inline README.md:1]: I think we should update the version in the README to 3.10 so it matches the workflow.

**Review by acidcoke — COMMENTED** [review]
This works and looks great. I just made some small suggestions. 

I can also change them when merging, if you want.

**Comment by willwade** [comment]
looks good to me now. Want me to resolve conflicts?

**Comment by acidcoke** [comment]
Looks perfect now. Thanks, but I'd like it do do myself.

**Pull request merged**